seqr/views/apis/family_api.py

```diff
@@ -80,15 +80,26 @@ def family_page_data(request, family_guid):
     for individual_guid in outlier_individual_guids:
         response['individualsByGuid'][individual_guid]['hasRnaOutlierData'] = True
 
-    has_phentoype_score_indivs = PhenotypePrioritization.objects.filter(individual__family=family).values_list(
-        'individual__guid', flat=True)
-    for individual_guid in has_phentoype_score_indivs:
-        response['individualsByGuid'][individual_guid]['hasPhenotypeGeneScores'] = True
+    phentoype_prioritization_tools = (
+        PhenotypePrioritization.objects.filter(individual__family=family).values(
+            'individual__guid',
+            'individual__phenotypeprioritization__tool',
```

Comment:
So here you are getting the values from the `PhenotypePrioritization` model but then starting every field with `individual`, which is an antipattern. It seems like a better approach would be to use [djangos aggereation functionality ](https://docs.djangoproject.com/en/3.2/topics/db/aggregation/#values)to group these by individual and then get the values. In this case it would look something like
```
tools_agg = PhenotypePrioritization.objects.filter(individual__family=family).values('individual__guid').annotate(
  phenotypePrioritizationTools=ArrayAgg(JSONObject(tool='tool', createdDate='created_date''))
)
```